src/Cake.Common/Solution/Project/Properties/AssemblyInfoCreatorData.cs

```diff
@@ -123,7 +123,9 @@ private string AttributeValueToString(object value)
                 {
                     return stringValue == string.Empty
                         ? string.Empty
-                        : string.Concat("\"", value, "\"");
+                        : isRawValue
+                            ? value.ToString()
+                            : string.Concat("\"", value, "\"");
```

Comment:
Currently quotes with in string would break output, wondering if we should do something like this?
```suggestion
                            : string.Concat("\"", stringValue.Replace("\"", "\\\""), "\""));
```

Reply:
That part was existing behaviour, but I think you're right that that's a better way of doing it. I'll add that push again.